src/style.css

```diff
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: inline-block;
+  height: 12px;
+  width: 12px;
+  margin: 2px 6px 2px 2px;
+  background-image: url("/images/star.svg");
+  background-size: cover;
+}
+
+.stars__star:nth-child(n) {
+  background-image: url("/images/star.svg");
+}
+
+.stars--0 .stars__star:nth-child(-n+0) {
+  background-image: url("/images/star-active.svg");
+}
```

Comment:
this one is not needed, because this selector basically says "do not set `background-image: url("/images/star-active.svg")` to any `.stars__star`"

this is because `n` will start from 1,  but you have `-n`, so it will be `:nth-child(-1)`, and then `:nth-child(-2)` and so on. and since there can be no negative children, the styles inside this selector will never apply